server/utils.py

```diff
@@ -18,8 +21,50 @@ class Settings(BaseSettings):
 
 
 def get_input_topic(job_id: str):
-    return f"adala-input-{job_id}"
+    topic_name = f"adala-input-{job_id}"
+
+    ensure_topic(topic_name)
```

Comment:
Imo it might be better to only return the input topic name here, and leave it to the user to ensure the topic exists. Seems more "separation of concerns" to me.
I'm pretty on the fence though, since I can't think of a use case where you'd want the name of a non-existent topic (or not-yet-existent/no-longer-existent topic), so I'll leave it up to your judgment.